php/class-wp-cli.php

```diff
@@ -83,7 +83,7 @@ public static function get_cache() {
 			// 6 months, 300mb
 			$cache = new FileCache( $dir, 15552000, 314572800 );
 
-			// clean older files on shutdown with 1/50 probability
+			// Clean older files on shutdown with 1/50 probability
```

Comment:
WPCS wants comments to end with a period.
```suggestion
			// Clean older files on shutdown with 1/50 probability.
```